lib/chutney/TorNet.py

```diff
@@ -764,6 +764,19 @@ def _genRouterKey(self):
             sys.exit(1)
         self._env['fingerprint'] = fingerprint
 
+
+    def _setEd25519Id(self):
```

Comment:
This function isn't called from anywhere. Please call it at the end of _genRouterKey().

How do you know this function works? Have you checked the output?

Reply:
Yes, I am checking the output simultaneously and testing the function by writing the function in a separate file, with the path of the file where public key is stored and comparing the returned value with the output of command
 `cat 
chutney/net/nodes/000a/keys/ed25519_master_id_public_key | tail -c 32 | base64 | head -c -2` and  then searching for it in the `cached-microdescs` and `cached-microdescs.new` files and the value returned by the function is correct.

Reply:
Great!